rascaline-torch/src/calculator.cpp

```diff
@@ -0,0 +1,289 @@
+#include "rascaline/torch/calculator.hpp"
+#include "equistore/torch/tensor.hpp"
+#include "rascaline/torch/autograd.hpp"
+#include <c10/util/Exception.h>
+
+using namespace equistore_torch;
+using namespace rascaline_torch;
+
+// move a block created by rascaline to torch
+static equistore::TensorBlock block_to_torch(
```

Comment:
Shouldn't this be in equistore?

Reply:
We could move this to equistore-torch, but right now is a bit specific to rascaline, assuming f64 and contiguous data that can be kept alive just by keeping the TensorMap around. Once we have something like https://github.com/lab-cosmo/equistore/issues/56, we could support more use cases and move this code over there

Reply:
Okay fine for me.